services/monitors/monitors/schemas.go

```diff
@@ -61,20 +45,13 @@ var monitorSchema = map[string]*schema.Schema{
 		Type:     schema.TypeString,
 		Computed: true,
 	},
-	"statistics": {
-		Type:     schema.TypeList,
-		Computed: true,
-		Elem: &schema.Resource{
-			Schema: statisticsSchema,
-		},
-	},
-	"expression": {
+	"rule": {
 		Type:     schema.TypeList,
 		Required: true,
 		MinItems: 1,
 		MaxItems: 1,
 		Elem: &schema.Resource{
-			Schema: expressionSchema,
+			Schema: rule,
```

Comment:
Is it possible to keep the same naming convention as before?
```suggestion
			Schema: ruleSchema,
```